model/src/main/proto/profile.proto

```diff
@@ -72,3 +81,20 @@ message ProfileChooserUiModel {
     bool add_profile = 2;
   }
 }
+
+// Supported Story text sizes for user preference.
+enum StoryTextSize {
+  SMALL_TEXT_SIZE = 0;
+  MEDIUM_TEXT_SIZE = 1;
+  LARGE_TEXT_SIZE = 2;
+  EXTRA_LARGE_TEXT_SIZE =  3;
+}
+
+// Supported App language for user preference.
+enum AppLanguage {
```

Comment:
How are we deciding what goes in this list? From what I understand, this needs to match the languages the app itself supports. I suggest filing an issue tracking adding translations & adding a TODO here for that issue to track updating this list to match the supported translation list.

Reply:
added todo to track language translation.